### PR TITLE
Web 3010

### PR DESCRIPTION
**Pull request by boulch**
Make it work LINK object with target = "_blank" property in site menu.

### CODE REVIEW

File: cpskin/menu/browser/menu.py [hunk 1]
```diff
@@ -285,6 +285,8 @@ def _submenu(self, items, tabindex, menu_level=0, menu_classnames='', close_butt
         return submenu
 
     def _menuitem(self, item, tabindex, first=False, last=False, menu_level=0):
+        portal_membership = api.portal.get_tool('portal_membership')
+        can_edit = portal_membership.checkPermission('Modify portal content', self.context)
```
Comment (bsuttor) [inline cpskin/menu/browser/menu.py]: I think, "can edit" should be check on "menu item" object and not on context, it should be tested after these lines 

    if brain.Type == 'Link':
            obj = brain.getObject()

We would like to test if a user can edit menu and not context

File: cpskin/menu/browser/menu.py [hunk 2]
```diff
@@ -306,6 +308,11 @@ def _menuitem(self, item, tabindex, first=False, last=False, menu_level=0):
             desc = safe_unicode(brain.Description)
             url = brain.getPath()
             item_id = brain.getURL()[len(self.site_url):]
+            if brain.Type == 'Link' and not can_edit:
+                obj = brain.getObject()
+                if hasattr(obj, 'target_blank'):
+                    target = '_blank' if obj.target_blank is True else '_self'
```
Comment (bsuttor) [inline cpskin/menu/browser/menu.py]: else is not necessary, target is already equal to "_self"

**Comment by bsuttor** [comment]
@boulch can you also check why tests are no more passing on travis please

**Comment by boulch** [comment]
@bsuttor 
Now I use api.user to check perm instead of portal_membership
                current_user = api.user.get_current()
                can_edit = current_user.has_permission('Edit', brain)

I take a look to tests

**Pull request merged**